Lib/test/test_dis.py

```diff
@@ -390,6 +398,42 @@ def foo(x):
        _h.__code__.co_firstlineno + 3,
 )
 
+
+
+@contextlib.contextmanager
+def ignore_coroutine_never_awaited_warnings():
```

Comment:
@1st1 Good point, although `coro.close()` is the appropriate method call that won't raise `StopIteration` in the test case.

I'll add a comment to the relevant line in the test.